### PR TITLE
update funding, add author randomization

### PR DESCRIPTION
**Pull request by jharenza**
This PR:
- Sets appropriate funding acquisition contribution
- Adds bit on author randomization
- Adds Jenn Mason's middle initial

Let me know if you think the randomization part sounds OK

### CODE REVIEW

File: content/05.administrative.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Finally, we would like to thank Yuanchao Zhang and Eric Wafula for adding to the
 |---|---|{% for author in manubot.authors %}
 |{{author.name}}|{% if author.contributions is string %}{{author.contributions}}{% else %}{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}{% endif %}|{% endfor %}
 
+Note: Except for the first and last four authors, authorship order was determined as follows.
+Authors who contributed to the OpenPBTA code base were listed based on number of modules to which that individual contributed and randomized in the case of a tie.
+The remaining order of authors was randomized.
+Code for determining authorship order can be found in the `count-contributions` module of the OpenPBTA analysis repository.
```
Comment (jaclyn-taroni) [inline content/05.administrative.md:21]: This tense makes more sense to me for talking about current order (but now how it was set)

```suggestion
Except for the first and last four authors, authorship order was determined as follows: Authors who contributed to the OpenPBTA code base are listed based on number of modules included in the manuscript to which that individual contributed and, in the case of ties, a random order is used. All remaining authors are then listed in a random order.

Code for determining authorship order can be found in the `count-contributions` module of the OpenPBTA analysis repository.
```

File: content/05.administrative.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-This project has been funded in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. 75N91019D00024, Task Order No. 75N91020F00003 (JLR, ACR, APH). 
+This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Childhood Cancer Data Lab (CSG), ALSF Young Investigator Award (JLR), ALSF Catalyst Award (JLR, ACR, PBS), ALSF Catalyst Award (SJS), ALSF CCDL Postdoctoral Training Grant (SMF), Children’s Hospital of Philadelphia Division of Neurosurgery (PBS and ACR), the Australian Government, Department of Education (APH), the St. Anna Kinderkrebsforschung, Austria (ARP), the Mildred Scheel Early Career Center  Dresden P2, funded by the German Cancer Aid (ARP), and NIH Grants 3P30 CA016520-44S5 (ACR), U2C HL138346-03 (ACR, APH), U24 CA220457-03 (ACR), K12GM081259 (SMF), R03-CA23036 (SJD), and NIH Contract No. HHSN261200800001E (SJD).
```
Comment (jaclyn-taroni) [inline content/05.administrative.md:4]: Okay, so in this scenario this is akin to the mention of "Intramural Research Program of the Division of Cancer Epidemiology and Genetics of the National Cancer Institute" where funds supported the work, just not via a grant.

**Review by jaclyn-taroni — APPROVED** [review]
I had some suggestions about the random order wording, but nothing super tightly held.

File: content/05.administrative.md (hunk 2 above)
Comment (jharenza) [reply content/05.administrative.md:4]: ok gotcha - @cgreene what type of ALSF grant did you have? I can add that separate of the Data Lab support

Comment (jaclyn-taroni) [reply content/05.administrative.md]: This should be fine as is.

**Comment by jaclyn-taroni** [comment]
I am going to revert https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/344/commits/55ab4971b3433fc32fa2e49828c5988331e95025, I was just checking my understanding with my comment.

**Pull request merged**